### DISCUSSION TITLE
我想输入中文应该怎么做？ What should I do to input Chinese？

### DISCUSSION THREAD

**Discussion by bright-zxl** [neutrinolabs/xrdp#2289] [Q&A] [answered]
朋友们，我想输入中文应该怎么做？
What should I do to input Chinese？

**Comment by matt335672** [comment]
Hi @bright-zxl 

I don't understand your question exactly. What are you trying to do?

**Comment by unstabler** [comment, marked as answer]
you can install input method like fcitx, ibus ... to input CJK languages.
to install fcitx5 with input engine (like pinyin ...), see https://wiki.archlinux.org/title/Fcitx5
